07LessonSeven/homework/index.html

```diff
@@ -5,6 +5,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+<header>
+  <h1>Professional Portal</h1>
+</header>
+  <nav>
+<div>
+<a href=""><span>home</span></a>
+<a href=""><span>Companies</span></a>
+<a href=""><span>Positions</span> </a>
+</div>
+
+  </nav>
+  <section>
+  <div class="box1">
+    <h2>Hire an employee</h2>
+   <p>
+     Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.
+  <button class="button" name="button">Sign up as company</button>
+   </p>
+
+  </div>
+  <div class="box2">
+   <h2>Looking for a job</h2>
+  <p>
+     Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.
+  <button class="button" name="button">Sign up as employee</button>
+   </p>
+ </div>
+</section>
 
 </body>
 </html>
```

Comment:
Thanks lugo. I've been ridiculous busy with work since they Merged my company I've had meetings and traveling from here to Austin it's been crazy but we're finally done and I'm back to business Sr.

Sent from my iPhone

> On Aug 16, 2016, at 12:51 PM, Justin Lugo <notifications@github.com> wrote:
> 
> In 07LessonSeven/homework/index.html:
> 
> > +  <div class="box1">
> > +    <h2>Hire an employee</h2>
> > +   <p>
> > +     Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.
> > +  <button class="button" name="button">Sign up as company</button>
> > +   </p>
> > +
> > +  </div>
> > +  <div class="box2">
> > +   <h2>Looking for a job</h2>
> > +  <p>
> > +     Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.
> > +  <button class="button" name="button">Sign up as employee</button>
> > +   </p>
> > + </div>
> > +</section>
> >  
> >  </body>
> >  </html>
> The site itself looks good. The main header (Professional Portal) needs to line up with the navigation links and the main content. Also, focus on code readability in your next assignment. Keep working hard Larry!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>